buttercms.go

```diff
@@ -1,17 +1,19 @@
 package ButterCMS
 
 import (
-	"strings"
 	"encoding/json"
 	"errors"
 	"io/ioutil"
 	"net/http"
+	"strings"
 )
 
 var authToken string
 
+var preview bool
+
 const (
-	VERSION = "2.3.0"
+	VERSION      = "2.3.0"
```

Comment:
What does `VERSION` control / impact? It doesn't seem to correspond with our tag versions: https://github.com/ButterCMS/buttercms-go/tags

Reply:
@jakelumetta  It is version of GO `req.Header.Set("X-Butter-Client", "Go/"+VERSION)`. I did not change the version value. Just linter added spacing.